### ISSUE TITLE
SIGNATURE VERIFICATION FAILED

### ISSUE THREAD

**Issue by sararita93** [micahflee/torbrowser-launcher#570]
Error Code: 
110775B5D101FB36BC6C911BEB774491D9FF06E2: Key expired

**Comment by Vertical-Card** [comment]
In the morning I ran into the same problem, I solved it this way: 
gpg --homedir "$HOME/.local/share/torbrowser/gnupg_homedir/" --refresh-keys --keyserver pgp.mit.edu

**Comment by rogers0** [comment]
@Vertical-Card 
I already reported this key expiration issue to upstream, and they're fixing it.
- https://gitlab.torproject.org/tpo/tpa/team/-/issues/40299

> gpg --homedir "$HOME/.local/share/torbrowser/gnupg_homedir/" --refresh-keys --keyserver pgp.mit.edu

actually it's recommend to use command below as workaround now:
```
gpg --homedir ~/.local/share/torbrowser/gnupg_homedir --keyserver hkps://keys.openpgp.org --refresh-keys 0x4E2C6E8793298290
```

**Comment by ZurbanPL** [comment]
How to workaround with this command?

**Comment by micahflee** [comment]
Fixed in torbrowser-launcher 0.3.4
